docs/developers/node-tutorial.mdx

```diff
@@ -749,8 +749,8 @@ celestia light start --core.ip <ip-address> --keyring.accname <key_name> --gatew
 ````
 
 Once you start the Light Node, a wallet key will be generated for you.
-You will need to fund that address with Arabica Devnet tokens to pay
-for PayForData transactions.
+You will need to fund that address with Mocha Testnet or Arabica Devnet
+tokens to pay for `PayForData` transactions.
```

Comment:
This doesn't seem true for both. One requires PFD, the other PFB

Reply:
2d4a476bf7f96b1a30a321e8ed011dfc0eb2f466